source/includes/mongosync-and-oidc.rst

```diff
@@ -0,0 +1,5 @@
+Starting in 1.8.1, you can use ``mongosync`` with Workload Identity
+Federation to authenticate connections to MongoDB clusters running
+on Microsoft Azure and Google Cloud Platform. Workload Identity
+Federation uses OpenID Connect (OIDC). OIDC is a protocol that allows
+users to authenticate credentials with an external identity provider.
```

Comment:
Patrick & I chatted about this feature yesterday. One of my takeaways from that conversation was that this feature targets users who know most of what they need to know—probably more than I do at this point, honestly—to make it work.

In this light, describing what OIDC is in the abstract seems superfluous here. Ultimately it’s more a question of documentation style, but FWIW.